scripts/install_tethys.bat

```diff
@@ -0,0 +1,61 @@
+# WARNING this is completely untested!!!!
+# Just some snippits to get started
+
+# TODO make these commandline options
+SET TETHYS_HOME="C:\tethys"
+SET TETHYS_PORT=8000
+SET TETHYS_DB_PORT=5435
+SET CONDA_HOME="%TETHYS_HOME%\miniconda"
+SET BRANCH=dev
+
+sudo mkdir -p %TETHYS_HOME%
+
+bitsadmin.exe /transfer "JobName" https://repo.continuum.io/miniconda/Miniconda3-latest-Windows-x86_64.exe "%TETHYS_HOME%\miniconda.exe"
+start /wait "" %TETHYS_HOME%\miniconda.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%CONDA_HOME%
+
+# clone Tethys repo
+conda install --yes git
+git clone https://github.com/tethysplatform/tethys "%TETHYS_HOME%/src"
+cd "%TETHYS_HOME%/src"
+git checkout %BRANCH%
+
+# create conda env and install Tethys
+conda env create -f tethys_conda_env.yml
+. activate tethys
+python setup.py develop
+tethys gen settings -d "%TETHYS_HOME%/src/tethys_apps"
+
+# Setup local database
+conda install --yes -c conda-forge postgresql
+initdb  -U postgres -D "%TETHYS_HOME%/psql/data"
```

Comment:
Here is a command to open a port on Windows that probably will be needed when someone get's to this:

```
netsh advfirewall firewall add rule name="postgresql" dir=in action=allow protocol=TCP localport=%TETHYS_DB_PORT%
```